capillary-services/PARTNER-PRICE.md

```diff
@@ -0,0 +1,441 @@
+# Capillary Services Endpoints
+
+## Website -Price and Stock -API
+
+### Fields/Data Model
+
+
+**uuid**
+
+- Data type: uuid, Must
+- Description: Product ID, could be product_uuid or byprice_uuid
+- Validation: Should be a proper longitude
+- Examples: f6bafc68-06b3-4c09-b044-978a0a7404f6 or 1d73aa2f-beee-426f-bbcd-f69556aaf873
+
+**universal_product_id**
+
+- Data type: ean or gtin, Must
+- Description: Universal Product ID, could be ean or gtin
+- Validation: Should be a proper longitude
+- Examples: 7501409201041 or 7501299330265
+
+**latitude**
+
+- Data type: float, Must
+- Description: Latitude of center point
+- Validation: Should be a proper latitude
+- Examples: 19.308533 or 19.336679
+
+**longitude**
+
+- Data type: float, Must
+- Description: Longitude of center point
+- Validation: Should be a proper longitude
+- Examples: -99.125781 or -99.112842
+
+**radius**
+
+- Data type: int, optional but strongly recommended
+- Description: Radius of area to get prices from
+- Validation: Should be an integer from 1 to 100
+- Examples: 10 or 30
+
+**zip_code**
+
+- Data type: int, optional but strongly recommended
+- Description: Zip code to get prices from
+- Validation: Should be an integer valid zip_code of 5 digits
+- Examples: 06600 or 04920
+
+
+#### Request
+
+```
+GET /capillary/website/item/get?uuid=6202a24c-c821-4f7a-bd96-f3b68bdef4c6&latitude=19.4326764&longitude=-99.1336339&radius=1 HTTP/1.1
```

Comment:
This is not needed , it is only for internal use

Reply:
removed